src/NUnitEngine/nunit.engine/Runners/MasterTestRunner.cs

```diff
@@ -296,11 +296,8 @@ private void ExpandProjects(TestPackage package)
         {
             if (package == null) throw new ArgumentNullException("package");
 
-            string packageName = package.FullName;
-            if (File.Exists(packageName) && _projectService.CanLoadFrom(packageName))
-            {
```

Comment:
Presume this change is for the tests. Did we check this that still behaves nicely if the `packageName` doesn't exist as a file in 'production'?

Reply:
It was a redundant check because the ProjectService simply ignores package file names that don't exist. The check in this location was forcing me to create any file names I want to pass to the faike project loader so I removed it.

Reply:
Great - I’m happy so long as it’s checked elsewhere! 😄